spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -1460,7 +1462,13 @@ private void killOrRestart(boolean aborted) {
 				}
 			}
 			else {
-				logger.info("Restarting " + this.consumer);
+				if (this.failedExclusive) {
+					getExclusiveConsumerExceptionLogger().logRestart(logger,
+							LogMessage.of(() -> "Restarting " + this.consumer));
+				}
+				else {
+					logger.info("Restarting " + this.consumer);
```

Comment:
Maybe this is moment when we can change that `logger` to the `LogAccessor` and use here a supplier variant?
Otherwise, consider to wrap it into `isInfoEnabled()`